src/test/java/com/google/testing/compile/GeneratingProcessor.java

```diff
@@ -29,9 +29,12 @@
 import javax.annotation.processing.RoundEnvironment;
 import javax.lang.model.SourceVersion;
 import javax.lang.model.element.TypeElement;
+import org.intellij.lang.annotations.Language;
 
 final class GeneratingProcessor extends AbstractProcessor {
   static final String GENERATED_CLASS_NAME = "Blah";
+
+  @Language("JAVA")
```

Comment:
I don't think these are worth adding. It's internal, plus for a single line file it seems minimally worth it.